frontend/src/components/ProjectList/components/Dataset/style.ts

```diff
@@ -15,10 +15,18 @@ export const Wrapper = styled.div`
 `;
 
 export const Name = styled.div`
-  ${fontStyle}
   width: ${LAYOUT.NAME}%;
   margin-right: ${LAYOUT.NAME_MARGIN}%;
   display: flex;
   align-items: center;
-  padding-left: 10px;
+  line-height: 1.3;
+  padding: 12px 0 12px 10px;
+`;
+
+export const NameChild = styled.div`
+  ${fontStyle};
+  display: block;
```

Comment:
ah `div` is `block` by default, so we can drop this 😄 !

Reply:
You are correct sir!